taxcalc/tests/test_policy.py

```diff
@@ -593,6 +535,7 @@ def test_pop_the_cap_reform():
     # specify a "pop the cap" reform that eliminates MTE cap in 2016
     reform = {'SS_Earnings_c': {2016: 9e99}}
     ppo.implement_reform(reform)
+    mte = ppo._SS_Earnings_c
```

Comment:
PT does not modify the same copy of the `_SS_Earnings_c` array. A completely new object is created. Thus, we need to re-initialize the `mte` variable.